### PR TITLE
[GAl-4256] Displaying Creators in NFT Token Page

### PR DESCRIPTION
**Pull request by Rohan-cp**
### Summary of Changes
1. deprecate ENABLED_CREATOR hard-coded variable
2. use this logic for NftDetailText
  
ownerIsHolder
 -> true -> display Owner
  -> false -> display null
  
ownerIsCreator
  -> true -> display Creator
  -> false -> look up contract's creator
      -> if exists -> display Creator
      -> else -> display null

### For creator being a gallery user

| Before                                     | After                                      |
| ------------------------------------------ | ------------------------------------------ |
| Screenshot 2023-10-26 at 1 55 17 AM | Screenshot 2023-10-26 at 1 14 12 AM |

### For creator only having contract
| Before                                     | After                                      |
| ------------------------------------------ | ------------------------------------------ |
| Screenshot 2023-10-23 at 9 17 56 PM | Screenshot 2023-10-23 at 9 12 26 PM |

### For owner also being the creator
| Before                                     | After                                      |
| ------------------------------------------ | ------------------------------------------ |
| Screenshot 2023-10-23 at 9 17 47 PM | Screenshot 2023-10-23 at 9 17 32 PM |

## Other improvements
Now handles ens name for contracts on nft detail and community view
| After                                     | After  (clicking on link)                                    |
| ------------------------------------------ | ------------------------------------------ |
| Screenshot 2023-11-01 at 5 06 18 AM | Screenshot 2023-11-01 at 5 06 53 AM |
### Edge Cases

List any common edge cases that you have considered and tested.

### Testing Steps

Provide steps on how the reviewer can test the changes.

### Checklist

Please make sure to review and check all of the following:

- [x] I've tested the changes and all tests pass.
- [ ] (if web) I've tested the changes on various desktop screen sizes to ensure responsiveness.
- [x] (if mobile) I've tested the changes on both light and dark modes.

### CODE REVIEW

File: apps/mobile/src/components/Community/CommunityMeta.tsx [hunk 1]
```diff
@@ -183,20 +183,18 @@ export function CommunityMeta({ communityRef, queryRef }: Props) {
...
           />
-          <LinkableAddress
-            chainAddressRef={community.contractAddress}
-            textStyle={{ color: colorScheme === 'light' ? colors.black[800] : colors.offWhite }}
-            font={{ family: 'ABCDiatype', weight: 'Bold' }}
-            eventElementId="Community Contract Address"
-            eventName="Community Contract Address Press"
-            eventContext={contexts.Community}
-          />
+          <View>
```
Comment (Robinnnnn) [inline apps/mobile/src/components/Community/CommunityMeta.tsx:186]: nit: does this need to be wrapped in a View?

File: apps/mobile/src/components/EnsOrAddress.tsx [hunk 2]
```diff
@@ -30,20 +35,43 @@ const EnsName = ({ chainAddressRef, eventContext }: EnsNameProps) => {
...
-  const { data } = useSWR(
-    chainAddressRef
-      ? `https://api.ensideas.com/ens/resolve/${encodeURIComponent(address.address.toLowerCase())}`
-      : null
-  );
+  const [addressData, setAddressData] = useState<Address | null>(null);
+  const { colorScheme } = useColorScheme();
+  const [error, setError] = useState<Error | null>(null);
+
+  useEffect(() => {
```
Comment (Robinnnnn) [inline apps/mobile/src/components/EnsOrAddress.tsx:42]: woooo awesome!

**Review by Robinnnnn — APPROVED** [review]

File: apps/mobile/src/components/Community/CommunityMeta.tsx (hunk 1 above)
Comment (Rohan-cp) [reply apps/mobile/src/components/Community/CommunityMeta.tsx:186]: yeah, I added it because the space-x-1 from the parent's styling wasn't working without it
